exit status: 1

stdout = """
def error: Type = {x: Int, y: Bool};
types of expressions:
span   | source                                   | type
-------|------------------------------------------|-----------------------------
12..36 | {x: Int, y: Bool, x: ()}                 | Type
16..19 | Int                                      | Type
24..28 | Bool                                     | Type
"""

stderr = """
error: duplicate field `x` in record type
  ┌─ tests/elab/errors/record_type/duplicate_fields.pion:1:31
  │
1 │ def error = {x: Int, y: Bool, x: ()};
  │              -                ^ duplicate field
  │              │                 
  │              first field

aborting due to 1 previous error
"""
